modules/tinymce/src/core/main/ts/newline/InsertLi.ts

```diff
@@ -64,6 +70,10 @@ const insert = (editor: Editor, createNewBlock: (name: string) => Element, conta
   }
 
   let newBlock = createNewBlock(newBlockName);
+  const parentBlockStyles = isListItem(parentBlock) ? getStyles(parentBlock) : undefined;
```

Comment:
If the get styles return an empty string if there is no styles or using an object instead of a string then you should be able to do this in one line like this.
```
const newBlock = isListItem(parentBlock) ? createNewBlock(newBlockName, { style: getStyles(parentBlock) }) ? createNewBlock(blockName);
```
But I guess it's a matter of style. I try to avoid using local let statements for mutatin or if branching if possible.

Reply:
changed